### PR TITLE
Grades are Sorted in Subject Screen

### PR DESCRIPTION
**Pull request by mujafar7**

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -10,10 +10,21 @@ export enum COURSES {
 }
 
 export enum SL_GRADES {
```
Comment (Sutara-Skanda) [inline src/common/constants.ts:12]: rename the sl_Grades to Grades

File: src/common/constants.ts [hunk 2]
```diff
@@ -10,10 +10,21 @@ export enum COURSES {
...
+  BELOWGRADE1 = "Below Grade 1",
   GRADE1 = "Grade 1",
   GRADE2 = "Grade 2",
+  GRADE3 = "Grade 3",
+  ABOVEGRADE3 = "Above Grade 3",
 }
 
+export const SL_GRADES_NUMERIC = {
```
Comment (Sutara-Skanda) [inline src/common/constants.ts:20]: same here also

**Review by Sutara-Skanda — CHANGES_REQUESTED** [review]

File: src/common/constants.ts [hunk 3]
```diff
@@ -9,11 +9,22 @@ export enum COURSES {
   MATHS = "maths",
 }
 
-export enum SL_GRADES {
+export enum GRADES {
+  BELOWGRADE1 = "Below Grade 1",
```
Comment (bhanugummana) [inline src/common/constants.ts:13]: Use Firebase doc ids instead of names...

**Review by bhanugummana — CHANGES_REQUESTED** [review]

**Comment by mujafar7** [comment]
sorting grade ss

**Review by bhanugummana — APPROVED** [review]

**Review by Sutara-Skanda — APPROVED** [review]

**Pull request closed without merging**